internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -736,37 +736,29 @@ func (mc *mac) waitPush(info *streamInfo, pushInfo *pushStatus) {
 	}
 }
 
-type WebhookMessage struct {
-	ID        string                 `json:"id,omitempty"`
-	Event     string                 `json:"event,omitempty"`
-	CreatedAt int64                  `json:"createdAt,omitempty"`
-	UserID    string                 `json:"userId,omitempty"`
-	StreamID  string                 `json:"streamId,omitempty"`
-	Payload   map[string]interface{} `json:"payload,omitempty"`
-}
-
 func (mc *mac) emitWebhookEvent(info *streamInfo, pushInfo *pushStatus, event string) {
 	if mc.producer == nil {
 		return
 	}
-	wm := WebhookMessage{
-		ID:        uuid.NewString(),
-		Event:     event,
-		CreatedAt: time.Now().UnixNano() / int64(time.Millisecond),
-		UserID:    info.stream.UserID,
-		StreamID:  info.stream.ID,
-		Payload: map[string]interface{}{
-			"target": map[string]interface{}{
-				"id":      pushInfo.target.ID,
-				"name":    pushInfo.target.Name,
-				"profile": pushInfo.profile,
-			},
+
+	payload := data.MultistreamWebhookPayload{
```

Comment:
It would be really nice to add some information about what is the error here, in case of a `multistream.error` event. Otherwise, all we'll be able to say is that the Multistream target hasn't connected, but no info about why.

@darkdarkdragon do you think we can grab that information from mist somehow?

Reply:
Talked on planning about this. From a product perspective, let's stick with what we have for this initial launch and have some directions about what to do in the docs instead. We can work on surfacing these error messages on the next iteration.